cardano-cli/test/data/golden/shelley/transaction-view.out

```diff
@@ -66,7 +66,7 @@ update proposal:
       pool influence: 82/83
       pool registration deposit: 83 Lovelace
       pool retirement epoch boundary: 84
-      protocol version: '85.86'
```

Comment:
What is this change?  How did we go from `85` to `8`?

Reply:
Very easy. Protocol version `85` is no longer possible to construct. Currently the highest possible major protocol version is `10`.
* `8` is current version in Babbage
* `9` will be `Conway`
* `10` is the version when upgrade `Conway`

We enforce this validity with `Version` type.